lib/build-all-ng.sh

```diff
@@ -72,7 +72,7 @@ pack_upload ()
 	# pack and upload to server or just pack
 
 	display_alert "Signing" "Please wait!" "info"
-	local version="Armbian_${REVISION}_${BOARD^}_${RELEASE}_${BRANCH}_${VER/-$LINUXFAMILY/}${DESKTOP_ENVIRONMENT:+_$DESKTOP_ENVIRONMENT}"
+	local version="${VENDOR}_${REVISION}_${BOARD^}_${RELEASE}_${BRANCH}_${VER/-$LINUXFAMILY/}${DESKTOP_ENVIRONMENT:+_$DESKTOP_ENVIRONMENT}"
```

Comment:
New variables need to be added here https://github.com/armbian/build/blob/master/lib/build-all-ng.sh#L47-L67

Reply:
Should I add $MAINTAINER and $MAINTAINERMAIL too?

Reply:
Oh, I might be misleading you. If variable is global, then this is not needed. And its needed only when building things in parallel. Leave as is.

Reply:
After much reflection, it is better to keep such settings in user local config. I added "unset" as you suggested.